captureData.js

```diff
@@ -0,0 +1,47 @@
+function captureData() {
```

Comment:
We likely want this as an object too, shall I approve like this and we can clean up?

```
const captureData {
  init() {
    ...
  },
  otherMethod() {
  }
}
```